index.html

```diff
@@ -2,15 +2,120 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <meta charset="utf-8">
+    <title>Great Idea - Responsive I</title>
 
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 
 <body>
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+<div class="nav-holder">
+    <nav>
+        <hr class="nav-line">
+        <a href="services.html">Services</a>
+        <hr class="nav-line">
+        <a href="">Product</a>
+        <hr class="nav-line">
+        <a href="">Vision</a>
+        <hr class="nav-line">
+        <a href="">Features</a>
+        <hr class="nav-line">
+        <a href="">About</a>
+        <hr class="nav-line">
+        <a href="">Contact</a>
+        <hr class="nav-line">
+    </nav>
+        <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> </a>
+</div>
+
+<header>
+    <div id="header-left">
+        <h1>
+            Innovation
+            <br/>
+            On
+            <br/>
+            Demand
+        </h1>
+        <button>
+            Get Started
+        </button>
+    </div>
+
+    <img id="code-snippet" src="img/header-img.png" alt="Image of a code snippet.">
+</header>
+
+<section class="text-block-holder">
+<hr class="cross-line">
+</section>
```

Comment:
purely preference but you don't ever really need a tag to wrap a single element / tag